database.php

```
@@ -0,0 +1,7 @@
+<?php
```

It's common practice to suppress errors using `@` with functions like `mysqli_connect`. Instead, handle them gracefully by setting error reporting levels and handling exceptions.